crowbar_framework/app/views/nodes/_buttons_power.html.haml

```diff
@@ -1,10 +1,22 @@
-- if @node.admin?
-  - if @node.bmc_set?
-    = link_to "Identify", hit_node_path(@node.handle, "identify"), :class => "btn btn-default", "data-remote" => true
-- else
-  = link_to "Reboot", hit_node_path(@node.handle, "reboot"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
-  = link_to "Shutdown", hit_node_path(@node.handle, "shutdown"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
-
-  - if @node.bmc_set?
-    = link_to "Power on", hit_node_path(@node.handle, "poweron"), :class => "btn btn-default", "data-remote" => true, "data-confirm" => t("are_you_sure")
-    = link_to "Identify", hit_node_path(@node.handle, "identify"), :class => "btn btn-default", "data-remote" => true
+- if @node.bmc_set?
+  - unless @node.admin?
+    = link_to t(".poweron"), hit_node_path(@node.handle, "poweron"), class: "btn btn-default", remote: true, confirm: t("are_you_sure")
```

Comment:
Wait, that one should be in the dropdown menu since the menu is about power actions :-)